include/Dialect/LWE/IR/LWEAttributes.td

```diff
@@ -230,4 +230,13 @@ def RLWE_InverseCanonicalEmbeddingEncoding
   }];
 }
 
+def LWE_LWEParams : AttrDef<LWE_Dialect, "LWEParams"> {
+  let mnemonic = "lwe_params";
+
+  let parameters = (ins "APInt": $cmod, "unsigned":$dimension);
```

Comment:
This may need the same trick we had as for Poly, where MLIR will try to compare APInts in the uniquer, and fail if they have different bit widths.

I wonder if we should bring up this to the MLIR folks more generally.

Reply:
Done